lib/github_service/issue.rb

```diff
@@ -42,12 +42,24 @@ def author
       user.login
     end
 
-    private
+    def organization_name
+      fq_repo_name.split("/")[0]
+    end
+
+    def repo_name
+      fq_repo_name.split("/")[1]
+    end
```

Comment:
There's a bit of proliferation of this logic, as it's also present in the Branch model.  I wonder if there's a nice way to DRY it up?  Not necessary for this PR, just wanted to put it out there.